src/test/java/uk/gov/companieshouse/overseasentitiesapi/controller/FilingsControllerTest.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.companieshouse.overseasentitiesapi.controller;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import org.springframework.http.HttpStatus;
+import uk.gov.companieshouse.api.model.filinggenerator.FilingApi;
+import uk.gov.companieshouse.api.model.transaction.Transaction;
+import uk.gov.companieshouse.overseasentitiesapi.exception.ServiceException;
+import uk.gov.companieshouse.overseasentitiesapi.exception.SubmissionNotFoundException;
+import uk.gov.companieshouse.overseasentitiesapi.service.FilingsService;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class FilingsControllerTest {
+
+    private static final String OVERSEAS_ENTITY_ID = "abc123";
+    private static final String TRANSACTION_ID = "def456";
+    private static final String ERIC_REQUEST_ID = "XaBcDeF12345";
+
+    private Transaction transaction;
+
+    @InjectMocks
+    private FilingsController filingsController;
+
+    @Mock
+    private FilingsService filingsService;
+
+    @Test
+    void getFiling() throws ServiceException, SubmissionNotFoundException {
```

Comment:
Suggest renaming these test methods - see comments in test class below. Up to you.